src/components/shared/pricing/PricingSectionWrapper.tsx

```diff
@@ -80,11 +80,26 @@ export const PricingSectionWrapper = ({
       <div className="flex flex-col items-center gap-10">
         <div className="flex flex-col items-center gap-4 text-center" id="priceTable">
           <Header maxWidth="max-w-4xl">{title()}</Header>
-          <p className="w-full max-w-3xl text-center text-xl text-gray-80">
-            {!isIndividual
-              ? `${hideBusinessCards ? textContent.businessDescription : textContent.businessDescription2}`
-              : `${textContent.planDescription}`}
-          </p>
+           <p className="w-full max-w-3xl text-center text-2xl !leading-tight text-regular text-gray-100">
```

Comment:
Pass a prop called CustomDescription?: ReactNode or smth like that and Make the comparison with that. Example:
````

CustomDescription ?? !isIndividual
              ? `${hideBusinessCards ? textContent.businessDescription : textContent.businessDescription2}`
              : `${textContent.planDescription}`
`````

In this way, we have the standard format for all LPs with the option to pass a custom component for a specific LP. We also decouple unnecessary logic in this wrapper.